examples/web-ui/material/main.smash

```diff
@@ -0,0 +1,20 @@
+// Sample functional tests of Material Design (https://material.io/)
+// Shows how to test a website that has a different navigation for desktop and mobile
+// Steps are implemented in other .smash files
+
+// Run these tests by running `smashtest --recursive` in this directory
+
+Open Chrome
+
+    Set Desktop
+    Set Mobile
```

Comment:
I want to encourage using the "On" pattern, as described on https://smashtest.io/language/functions
I know that's contrary to what I said on gitter, but I feel this standard is best. Please let me know what you think!